s3_file_uploads/models.py

```diff
@@ -22,6 +26,13 @@ class UploadedFile(ConcurrentTransitionMixin, models.Model):
     filename = models.CharField(max_length=255, blank=True)
     file_upload_state = FSMField(default=UPLOAD_STATES.NEW)
 
+    user = models.ForeignKey(
+        USER_MODEL,
+        related_name='uploaded_files',
+        on_delete=models.CASCADE,
+        null=True
```

Comment:
Should we accept null? Not sure what to do with the existing ones, but wondering if it is safe enough like this.

Reply:
Discussed with @lCharlie123l - nullable is ok as the restriction should be forced to occur in the app via the new field

Reply:
also you may not want the files to be deleted when you delete a user in your app so this seemed a sensible default - however the `on_delete` should be `SET_NULL` rather than cascade here

Reply:
Set to null as requested